contracts/EpochTokenLocker.sol

```diff
@@ -105,8 +105,8 @@ contract EpochTokenLocker {
         return balanceStates[user][token].pendingWithdraws.stateIndex;
     }
 
-    function getCurrentStateIndex() public view returns(uint32) {
-        return uint32(now / BATCH_TIME);
+    function getCurrentStateIndex() public view returns(uint) {
+        return uint(now / BATCH_TIME);
```

Comment:
is this cast needed?

Reply:
Probably not anymore. Used to be.. Will remove.